### ISSUE TITLE
Dependency on a non-existent service "request"

### ISSUE THREAD

**Issue by benr77** [DZunke/FeatureFlagsBundle#24]
Installing v4.1 on Symfony 4.4 gives me the following error when I clear the cache:

```
In CheckExceptionOnInvalidReferenceBehaviorPass.php line 86:
                                                                                                       
  The service "dz.feature_flags.conditions_bag" has a dependency on a non-existent service "request". 
```

I can't see where the reference to "request" is coming from. Any ideas?
Thanks

**Comment by benr77** [comment]
OK, found it. The service configuration was injecting `@request` which does not exist. Instead we should be injecting `@request_stack`.

Submitted PR #25 to fix this - simple tweak to service configuration only, compatible back to Symfony 2.4.

**Comment by DZunke** [comment]
Thanks @benr77 ... The Pull Request was merged and you'll find the fix in Version 4.2 👍

**Comment by benr77** [comment]
Thank you!
